specification/columns/commitmentdiscountunit.md

```diff
@@ -0,0 +1,43 @@
+# Commitment Discount Unit
+
+Commitment Discount Unit is the string value representing the provider-specified measurement unit that corresponds to [Commitment Discount Quantity](#commitmentdiscountquantity) of a [*commitment discount*](#glossary:commitment-discount). The CommitmentDiscountUnit column is only applicable to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount).
+
+The CommitmentDiscountUnit column adheres to the following requirements:
+
+* CommitmentDiscountUnit MUST be present in the billing data when the provider supports [*commitment discounts*](#glossary:commitment-discount).
+* CommitmentDiscountUnit MUST be of type String, and the units of measure used in CommitmentDiscountUnit SHOULD adhere to the values and format requirements specified in the [UnitFormat](#unitformat) attribute.
+* The CommitmentDiscountUnit MUST be the same across all *rows* where *CommitmentDiscountQuantity* has the same [*CommitmentDiscountId*](#commitmentdiscountid).
+* CommitmentDiscountUnit MUST NOT be null when *CommitmentDiscountId* is not null.
```

Comment:
when its a correction we're not asking for this definition as a MUST right? MUST when not a correction and SHOULD otherwise? @ijurica

Reply:
Correct.
As I recall, we planned to create a Git issue to address this as part of our Consistency review—specifically, to review and align how we specify Corrections across the specification.

In other columns (costs, quantities, units, etc.), we **allow null values for Corrections**. Additionally, we **didn’t address negative values explicitly**, even though it’s clear that quantities and costs can be negative only in the case of Correction charges and must be non-negative for non-Correction charges. . Please discuss (briefly), make a decision, and apply it consistently across the specification.